projects/angular/src/layout/_card.clarity.scss

```diff
@@ -14,10 +14,12 @@
 @include mixins.exports('card.clarity') {
   %card-border-appearance {
     // IE/OldEdge
```

Comment:
Maybe it's not the right time now, but all these are marked IE/OldEdge and probably we can just remove them.

Reply:
Yes, it's not the right time for this. Resolving